components/usage/pkg/apiv1/usage.go

```diff
@@ -5,13 +5,23 @@
 package apiv1
 
 import (
+	context "context"
+
 	v1 "github.com/gitpod-io/gitpod/usage-api/v1"
 )
 
+var _ v1.UsageServiceServer = (*UsageService)(nil)
+
 type UsageService struct {
 	v1.UnimplementedUsageServiceServer
 }
 
+func (us *UsageService) GetBilledUsage(ctx context.Context, in *v1.GetBilledUsageRequest) (*v1.GetBilledUsageResponse, error) {
+	// TODO(geropl) Dummy data for now
+	response := v1.GetBilledUsageResponse{}
+	return &response, nil
+}
```

Comment:
If you wanted a terser version, you could do

```suggestion
func (us *UsageService) GetBilledUsage(ctx context.Context, in *v1.GetBilledUsageRequest) (resp *v1.GetBilledUsageResponse, err error) {
    return resp, err 
}
```

Reply:
I know, but I generally prefer the explicit syntax - and room for the comment. 😉